website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,98 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### LDAP Auth Method Attributes
+
+The ldap auth method has the following additional attributes:
+
+- `state` - The state of the auth method; either inactive, active-private or
+  active-public
+
+- `start_tls` - (optional) If true, issues a StartTLS command after establishing
+  an unencrypted connection. Defaults to false.
+
+- `insecure_tls` - (optional) If true, skips LDAP server SSL certificate
+  validation - which is insecure and should be used with caution. Defaults to
+  false. 
+
+- `discover_dn` - (optional) If true, use anon bind to discover the bind DN
+  (Distinguished Name) of a user. Defaults to false.
+
+- `anon_group_search` - (optional) If true, use anon bind when performing LDAP
+  group searches. Defaults to false.
+
+- `upn_domain` - (optional) If set, the userPrincipalDomain used to construct
+  the UPN string for the authenticating user. The constructed UPN will appear as 
+  [username]@UPNDomain  Example: example.com, which will cause Boundary to
+  bind as username@example.com when authenticating the user.
+
+- `urls` - (required) The LDAP URLS that specify LDAP servers to connect to.
+  There must be at least one URL for each LDAP auth method. When attempting to
+  connect, the URLs are tried in the order specified. 
+
+- `user_dn` - (optional) If set, the base DN under which to perform user
+  search. Example: ou=Users,dc=example,dc=com
```

Comment:
```suggestion
  search. Example: ou=Users,dc=example,dc=com.
```

Reply:
So the period is required as part of the DN?  That's what I think when I read your suggestion and why I didn't add the period.

Reply:
I understand, in that case I really think we should surround the examples with backticks consistently, that would remove ambiguity and make the style consistent.

Reply:
I disagree.

Reply:
Fair enough 😄.